pkg/pull/signature.go

```diff
@@ -90,7 +90,7 @@ func verify(message, signature, publicKeyPEM []byte) error {
 
 func getMessageFromLicense(license *kotsv1beta1.License) ([]byte, error) {
 	// JSON marshaller will sort map keys automatically.
-	fields := map[string]string{
+	fields := map[string]interface{}{
```

Comment:
I think changing this to interface opens this up to programming mistakes.  You can now add an object of any type to this map.

Reply:
the license is signed with `licenseSequence` being an integer, that was the reason for changing this

Reply:
i guess i should change signing the license to use `licenseSequence` as string